backend/post-service/src/main/java/com/speakapp/postservice/dtos/LatestUserPostsPageGetDTO.java

```diff
@@ -0,0 +1,17 @@
+package com.speakapp.postservice.dtos;
+
+import java.util.List;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class LatestUserPostsPageGetDTO {
+  List<PostGetDTO> posts;
```

Comment:
Naturalniej by było zwracać tutaj Page skoro w repo też masz ściągany obiekt Page - napisałem na dole dokładniej o co mi chodzi

Reply:
UPDATE: przemyśłałbym to bardziej wgl w zasadzie to twoje podejście tutaj jest dosyć sensowne, page zawiera dużo innych informacji które są zbędne w tym przypadku, można się nad tym dłużej zastanowić dzisiaj wieczorem

Reply:
Pomyślimy